### PR TITLE
status: make instance status colorful

### PR DESCRIPTION
**Pull request by psergee**
Screenshot from 2022-12-06 12-24-44

### CODE REVIEW

**Comment by 0x501D** [comment]
non-standard unicode character. even pregmatapro doesn't draw it

**Comment by 0x501D** [comment]
NO_COLOR has no effect. We shouls support it.
https://no-color.org/

**Review by 0x501D — CHANGES_REQUESTED** [review]
I think it would be better to manipulate the color of the instance name than to use a unicode character, which is not supported by all fonts. And of course handle the NO_COLOR variable

**Comment by psergee** [comment]
> 
> 
> non-standard unicode character. even pragmatapro doesn't draw it

Sure, I know :)

File: cli/cmd/status.go [hunk 1]
```diff
@@ -39,7 +39,8 @@ func internalStatusModule(cmdCtx *cmdcontext.CmdCtx, args []string) error {
 	}
 
 	for _, run := range runningCtx.Instances {
-		log.Infof("%s: %s", run.InstName, running.Status(&run))
+		procStatus := running.Status(&run)
+		log.Infof("%c %s: %s", procStatus.Color, run.InstName, procStatus.Text)
```
Comment (AnaNek) [inline cli/cmd/status.go:43]: Maybe we should just use the colorful output using a standard [package color](https://pkg.go.dev/github.com/fatih/color)?

Comment (psergee) [reply cli/cmd/status.go:43]: This package is not standard.

**Review by 0x501D — APPROVED** [review]

**Review by AnaNek — APPROVED** [review]

**Pull request merged**